app/components/post_list/more_messages/more_messages.tsx

```diff
@@ -114,7 +119,8 @@ const MoreMessages = ({
     const [remaining, setRemaining] = useState(0);
     const underlayColor = useMemo(() => `hsl(${hexToHue(theme.buttonBg)}, 50%, 38%)`, [theme]);
     const top = useSharedValue(0);
-    const shownTop = isTablet || (isCRTEnabled && rootId) ? 5 : SHOWN_TOP;
+    const adjustedShownTop = SHOWN_TOP + (currentCallBarVisible ? CURRENT_CALL_BAR_HEIGHT : 0) + (joinCallBannerVisible ? JOIN_CALL_BAR_HEIGHT : 0);
+    const shownTop = isTablet || (isCRTEnabled && rootId) ? 5 : adjustedShownTop;
```

Comment:
@avinashlng1080 Can you take a quick look at this to see if it makes sense with what you fixed earlier today? Thanks!